Jobs.Fetcher.Facebook/Client/Fetcher.cs

```diff
@@ -328,7 +329,9 @@ public class Fetcher {
             }
             JObject row = FetchInsights(node["id"].ToString(), edge, new Range<DateTime>(startN - new TimeSpan(3, 0, 0, 0), upperLimit));
             JObject nobj;
-            if (edge.Transposed && row.SelectToken("data").Count() > 0) {
+            if (!row.HasValues) {
+                return null;
```

Comment:
At line 391 (now 394) this function gets called but there is no check against a possible null return value. Would it be redundant, in that case?